README.md

```diff
@@ -72,6 +72,29 @@ filed named `.gitkeep`. Finally, create individual locks by making an empty file
   retrying to acquire a lock or release a lock. The default is 10 seconds.
   Valid values: `60s`, `90m`, `1h`.
 
+* `prometheus_push_gateway`: *Optional.* URL to the prometheus push gateway that will
+  expose the following metrics for each execution of the `out` step:
+
+    * `resource_pool_execution_time`:
+      Total execution time of the current operation.
+      Labels: `pool, branch, operation`
+
+    * `resource_pool_num_available_locks`:
+      Number of available/unclaimed locks in the current pool.
+      Labels: `pool, branch`
+
+    * `resource_pool_num_claimed_locks`:
+      Number of claimed locks in the current pool.
+      Labels: `pool, branch`
+    
+    > Notes:
+    All metrics are computed after performing the operation.
```

Comment:
Not clear what 'after operation' means.

Reply:
operation: any of the 'lock foo', 'unlock foo', 'add lock foo', etc.
That means that, when you read the 'number of available locks' metrics, it reflects a value *after* performing a lock/unlock operation, not before.
If it was done before: you would never see the number of available lock drop back to zero, for example.

Reply:
Something like the following would make the text clearer:

> All metrics are computed after performing the specific lock operation.